lib/events/events.go

```diff
@@ -45,6 +43,11 @@ const (
 	ListenAddressesChanged
 	LoginAttempt
 
+	// Deprecated events
+
+	DeprecatedDevicePaused
```

Comment:
Likewise I see no reason in deprecating these events. Having more fine grained events is more useful to the consumer than just "the config changed. figure it out for yourself! ppppfft!" :)